### DISCUSSION TITLE
Reorganizing lists?

### DISCUSSION THREAD

**Discussion by TabFox-log** [community/community#78274] [Other Feature Feedback, Questions, & Ideas]
You should add an option to change to order of the lists, or auto sorting.

**Comment by mcfandw** [comment]
Agreed. I wish I could change the order repositories within my lists, because the current order appears to be simply when I added them to the list. Currently I'd have to remove them all and re-add them one-by-one to change the order.

**Reply by Agrament** [reply]
J
